### PR TITLE
Rewrite rpr test

### PR DESCRIPTION
**Pull request by AlexanderVeselov**

### CODE REVIEW

File: Rpr/WrapObject/ContextObject.cpp [hunk 1]
```diff
@@ -73,6 +73,7 @@ namespace
     { RPR_CONTEXT_TONE_MAPPING_REINHARD02_POST_SCALE,{ "tonemapping.reinhard02.postscale", "Reinhard postscale", RPR_PARAMETER_TYPE_FLOAT } },
     { RPR_CONTEXT_TONE_MAPPING_REINHARD02_BURN,{ "tonemapping.reinhard02.burn", "Reinhard burn", RPR_PARAMETER_TYPE_FLOAT } },
     { RPR_CONTEXT_MAX_RECURSION,{ "maxRecursion", "Ray trace depth", RPR_PARAMETER_TYPE_UINT } },
+    { RPR_CONTEXT_RANDOM_SEED,{ "randseed", "Random seed", RPR_PARAMETER_TYPE_UINT } },
```
Comment (Kvalme) [inline Rpr/WrapObject/ContextObject.cpp]: It's better to add value to the end not in the middle

File: RprTest/aov.h [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+class AovTest : public BasicTest
+{
+public:
+    void CreateFramebuffer() override
+    {
+        // Create Framebuffer
+        rpr_framebuffer_desc desc = { kOutputWidth, kOutputHeight };
+        rpr_framebuffer_format fmt = { 4, RPR_COMPONENT_TYPE_FLOAT32 };
+        ASSERT_EQ(rprContextCreateFrameBuffer(m_context, fmt, &desc, &m_framebuffer), RPR_SUCCESS);
```
Comment (Kvalme) [inline RprTest/aov.h:35]: Clean framebuffer before use or we clean it in rprContextCreateFrameBuffer?

File: RprTest/arithmetic.h [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+{
+    TestUnaryOperationTextureMap(RPR_MATERIAL_NODE_OP_SHUFFLE_ZWXY);
+}
+
+TEST_F(ArithmeticTest, Arithmetic_ShuffleWXYZ)
+{
+    TestUnaryOperationTextureMap(RPR_MATERIAL_NODE_OP_SHUFFLE_WXYZ);
+}
+
+// This doesn't work
```
Comment (Kvalme) [inline RprTest/arithmetic.h]: Please, check and provide more info

**Review by Kvalme — CHANGES_REQUESTED** [review]

File: RprTest/aov.h (hunk 2 above)
Comment (AlexanderVeselov) [reply RprTest/aov.h:35]: We clean it in BasicTest::Render() before taking render samples

File: RprTest/basic.h [hunk 4]
```diff
@@ -47,7 +47,7 @@ extern char** g_argv;
 class BasicTest : public ::testing::Test
 {
 public:
-    static std::uint32_t constexpr kRenderIterations = 32;
+    static std::uint32_t constexpr kRenderIterations = 256;
```
Comment (Kvalme) [inline RprTest/basic.h]: Please, made this change only for Bump and Normal map tests. Not for all tests

**Pull request closed without merging**